test/e2e-smoke/basic.spec.ts

```diff
@@ -19,9 +19,13 @@ test('basic app creation flow', async ({ page }) => {
 
   await page.click('[aria-label="Home"]');
 
-  const appCardSelector = `[role="article"]:has-text("${appName}")`;
+  const appRow = await page.locator(`[role="row"]`, {
+    has: await page.locator(`input[value="${appName}"]`),
```

Comment:
I'd prefer if we could keep the old locator. I have a hunch that there's an accessibility issue here. What happens if we add the `aria-readonly` to the input when it's not editable?

Reply:
Adding the `aria-readonly` still doesn't help use the original selector, which relied on a `has-text`; but now since the name resides in the `value` of an `input`, `has-text` fails to find the element.

I've modified it a bit to use the selector format instead of locators, and now seems like the tests pass for Firefox as well